MAUI/DataForm/Validation.md

```diff
@@ -15,6 +15,108 @@ The data form validates the data and user input to update the correct value in t
 
 The supported built in validations are as follows:
 
+#### IDataErrorInfo
+You can validate the data by implementing the [IDataErrorInfo](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.idataerrorinfo?view=net-6.0) interface in the data object class.
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : IDataErrorInfo
+{
+    public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
public event PropertyChangedEventHandler PropertyChanged;

Check is this needed, if not remove

Reply:
removed